playbooks/support/README.md

```diff
@@ -19,111 +50,167 @@ which is determined at least a month in advance (giving people ample time to mak
-  incidents at a glance.
-- Investigate the issue individually and as a pair.
-- Judge the severity (based on intuition or a helpful PM/business owner)
 - Fix if possible using [accumulated playbooks (wiki)](https://github.com/artsy/potential/wiki) 🔒.
 - If unable to fix using shared resources, contact the relevant [point-person](#point-people) for help and pair
-  with them on a fix. Be sure to contribute lessons back to shared documentation. **You are not expected to know the ins and outs of every system, so don't hesitate to involve the wider team.**
-- Point-people should only be contacted during off-hours for outages that the on-call engineers are unable to
-  diagnose.
+  with them on a fix. Be sure to contribute lessons back to shared documentation. **You are not expected to know
+  the ins and outs of every system, so don't hesitate to involve the wider team.**
+  - Point-people should only be contacted during off-hours for outages that the on-call engineers are unable to
+    diagnose.
 - Share any applicable work-arounds or talking points in the thread to unblock teammates or partners.
 - If you make an immediate fix but addressing the root cause requires further work, add the issue to the bug
   backlog so it can be prioritized accordingly.
-- If the issue was mis-reported and doesn't require a timely fix, surface to PM/add to backlog for tracking by the
-  appropriate team
 
-**Note:** You are not responsible for fixing all of the bugs that you find. Most problems that require a coding change should be addressed by the relevant team.
+**Note:** You are not responsible for fixing all of the bugs that you find. Most problems that require a coding
+change should be addressed by the relevant team.
 
-#### 3. Track incidents' status
+#### Communicate
 
-Incidents are recorded on [the incidents Trello board](https://trello.com/b/sZQ9qpVo/incidents), which provides
-visibility into what is being addressed and helps us track how often certain issues occur (so it's okay if
-something appears multiple times throughout the week). This board should only be edited by the on-call engineers
-and others helping out. It's not a replacement for reporting issues in slack, and tickets that require additional
-engineering work should be tracked in Jira.
+One of the on-call members handles internal and external communication for the incident.
 
-Tips for using the incidents Trello:
+- #incidents is for non-on-call people to raise possible incidents or follow up (questions, etc.) about existing
+  threads.
+- Resolution happens exclusively in the dedicated slack channel associated with the Jira Ops ticket.
+- Any meaningful updates or discoveries are recorded (by the assignee, by default) to the Jira Ops ticket.
+  Significant milestones (new incident, availability changes, incident resolved) are recorded this way.
+- Any incident with a severity of Critical should be communicated externally via the status page. Communication
```

Comment:
I'm not sure about the "critical" condition. Maybe just "Any incident with public impact?" We have some "pro" users like writers and partner admins who can self-serve availability information via the status page when issues impact them, even if not critical.